PYME/Acquire/Hardware/AndorNeo/AndorNeo.py

```diff
@@ -590,7 +590,8 @@ def GetFPS(self):
     def __getattr__(self, name):
         if name in self.noiseProps.keys():
             return self.noiseProps[name]
-        else:  raise AttributeError, name  # <<< DON'T FORGET THIS LINE !!
```

Comment:
The comment here is pretty crucial to carry over for anyone modifying the class

Reply:
I wasn't really worried someone would forget to raise an `AttributeError` inside of a `__getattr__` method. Why is there even a `__getattr__` method for the camera just to access  items of the noise_properties dict? Is it crucial to call `cam.ElectronsPerCount` instead of `cam.noise_properties['ElectronsPerCount']`? Either seems fine, it would just be nice to stick with one way of doing it.

Reply:
I've done it a few times, and in this case it generates really hard to diagnose errors (hence the warning).

Ultimately the whole attribute access stuff on cameras should be removed in favour of the `noise_properties` property, modulo a check to see if it's ever actually used in the current code.

Reply:
Actually the absence of `__getattr__()` in the much better maintained AndorZyla.py is good enough for me to go ahead with removal (there's no chance it'll be required in the Neo if it's not present in the Zyla as they share all UI code).

Reply:
Cool, thank you for checking the Zyla for that. I'll go ahead and remove the __getattr__ method then.